gravitee-apim-portal-webui/package.json

```diff
@@ -3,8 +3,8 @@
   "version": "0.0.0",
   "description": "Gravitee.io Portal",
   "scripts": {
-    "build": "ng build --aot",
-    "build:prod": "ng build --configuration production",
+    "build": "NODE_OPTIONS=--max_old_space_size=8192 ng build",
```

Comment:
😞  the portal take 6 go in memory or more .... and I didn't find out why ...

Reply:
:aa: